src/distributors/adoptopenjdk-models.ts

```diff
@@ -0,0 +1,45 @@
+export interface IReleaseVersion {
```

Comment:
May be we should include provider name to interfaces?
`IAdoptMajorVersions`
The same for `IRelease` -> `IAdoptAvailableVersions`

Reply:
I think it is not necessary because it is located in the file with provider names

Reply:
Done. Thank you.